show/reboot_cause.py

```diff
@@ -24,6 +26,98 @@ def read_reboot_cause_file():
     return reboot_cause_dict
 
 
+# Function to fetch reboot cause data from database
+def fetch_data_from_db(module_name, fetch_history=False, use_chassis_db=False):
+    prefix = 'REBOOT_CAUSE|'
+    if use_chassis_db:
+        try:
+            rdb = redis.Redis(host='redis_chassis.server', port=6380, decode_responses=True, db=CHASSIS_STATE_DB)
```

Comment:
Where does `redis_chassis.server` name come from?

Reply:
This is how the chassis state db is accessed in the modular chassis